# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libhid/no-doc.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Failed to load a DTD from the internet, which of course fails (intentionally)
inside the T2 build chroot sandbox.

  - Rene Rebe <rene@exactcode.de>

--- trunk/Makefile.am.vanilla	2008-08-20 11:31:41.000000000 +0200
+++ trunk/Makefile.am	2008-08-20 11:31:44.000000000 +0200
@@ -6,7 +6,7 @@
 if HAVE_SWIG
   MAYBE_PYTHONWRAPPER = swig
 endif
-SUBDIRS = hidparser src $(MAYBE_PYTHONWRAPPER) test doc
+SUBDIRS = hidparser src $(MAYBE_PYTHONWRAPPER) test
 
 dist_noinst_HEADERS = \
 	include/assert.h \
